e2e/tests/resource-page.spec.ts

```diff
@@ -0,0 +1,199 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Page, expect, test } from '@playwright/test';
+import { getResource, getResourceUrl } from '../utils';
+import { Resource } from '../../src/common/lib/types';
+
+test.describe('Resource page', async () => {
+  let page: Page;
+  let resourceUrl: string;
+  let resource: Resource;
+
+  const haukiUser = process.env.HAUKI_USER || '?';
```

Comment:
Could maybe default to `cypress_test_user` or something equivalent? 🤔

Reply:
Previously setting the correct HAUKI parameters for the e2e-tests was assigned to the one running the tests 👎🏻 Now the required variables are defined within the tests 👍🏻

`const testData = {
   HAUKI_USER: 'cypress_test_user',
   HAUKI_ORGANIZATION: 'test:cypress',
   HAUKI_SOURCE: 'test',
   HAUKI_RESOURCE: 'test:1',
 };`